packages/ui/src/components/molecules/Dialog/DialogBox.tsx

```diff
@@ -89,15 +91,31 @@ const DialogBoxBodyStyle = styled.div<DialogBoxUtilityProps>`
   justify-content: center;
   align-items: center;
   width: inherit;
-  padding-left: 40px;
-  padding-right: 40px;
-  padding-top: 32px;
-  padding-bottom: 32px;
   gap: 32px;
+  background-color: ${props => props.bgColor ?? 'inherit'};
+  /* overflow-y: auto;
+  overflow-x: hidden; Prevent horizontal scrollbar */
   ${flex}
   ${width}
   ${height}
   ${spacing}
+  // Set a fixed height
+ // height: 300px; // adjust this value as per your need
+  
+  // or set a max-height
+  max-height: calc(100vh - 200px); // adjust this value as per your need
```

Comment:
Do not modify the default css for existing components, it'll break the current UI

Reply:
done